test/unit/spec/common/cellComponents/filePathWidget-Spec.js

```diff
@@ -68,7 +69,7 @@ define([
         });
 
         beforeEach(function () {
-            this.bus = Runtime.make().bus();
+            this.bus = Bus.make();
```

Comment:
is this a more general switch that could be made?

Reply:
Yeah, I realized it late last night. This could probably be made in a number of other places.